lib/gitspindle/github.py

```diff
@@ -627,15 +625,14 @@ def deploy_keys(self, opts):
         """[<repo>]
            Lists all keys for a repo"""
         repo = self.repository(opts)
-        for key in repo.iter_keys():
-            ro = key._json_data['read_only'] and 'ro' or 'rw'
```

Comment:
Did this attribute go missing from the json data, or do we need to file a github3.py pr to add it as a proper attribute?

Reply:
I think the read_only attribute actually didn't go. I have restored this part.